sourcemash/frontend/templates/help.tpl

```diff
@@ -0,0 +1,91 @@
+<div class="help-fixed-action-btn">
+  <span class="btn-floating btn-large">?</span>
+</div>
+
+<div id="help-modal" class="modal">
+  <div class="modal-content section no-pad-bot">
+    <div class="container">
+      <div class="row">
+
+        <h2 class="col s12 light center header flow-text">Confused?</h2>
+        <h4 class="col s12 light center header flow-text">Let's get you back on track!</h4>
+        <p class="col s12 divider"><p>
+
+        <h5 class="col s12 light center header flow-text">Sourcemash categorizes your articles to help you get through your feeds faster.</h5>
+
+      </div>
+
+      <div class="row">
+        <div class="col s4">
+          <div class="center promo">
+            <a href="/#browse">
+              <i class="mdi-editor-insert-comment promo-icon"></i>
+            </a>
+          </div>
+        </div>
+        <div class="col s8">
+          <p class="promo-caption">Step 1: Source selection</p>
+          <p class="light">Choose the sources you want to read. Select from our list of feeds on the <a href="/#browse">Browse page</a>, or add your own using the form at the top of the page! Once you subscribe to a feed by toggling the "Subscribe" switch, it will appear in the "Feeds" tab in the left navigation menu and in the "Custom" section of the <a href="/#browse">Browse page.</a></p>
```

Comment:
Either<a href="/#browse"> browse</a> our list of feeds, or add your own using the form at the top of the <a href="/#browse">Browse Page</a>. When you see a feed you like, click to toggle to subscribe!